microsoft-edge/devtools-guide-chromium/dom/index.md

```diff
@@ -175,7 +176,7 @@ To use the HTML editor:
 
 1. In the DOM Tree, right-click the `<section id="edit-as-html-1">` node and select **Edit as HTML**. The HTML editor appears.
```

Comment:
```suggestion
1. In the DOM Tree, right-click the `<section id="edit-as-html-1">` node, and then select **Edit as HTML**. The HTML editor appears:
```
not sure if this is a lead-in for png, needing colon.
Rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/dom/?branch=pr-en-us-2884#edit-multiple-nodes-text-and-attributes